tests/test_parse_load_args.py

```diff
@@ -0,0 +1,116 @@
+import argparse
+import tempfile
+import unittest
+import uuid
+from contextlib import contextmanager
+from os import chdir, getcwd
+from pathlib import Path
+from typing import List, Optional
+
+import mkosi
+from mkosi.backend import Distribution, MkosiArgs, MkosiException
+
+
+def parse(argv: Optional[List[str]] = None) -> MkosiArgs:
+    return mkosi.load_args(mkosi.parse_args(argv)["default"])
+
+@contextmanager
+def change_dir(folder: str):
+    old_dir = getcwd()
+    try:
+        chdir(folder)
```

Comment:
Didn't we already have the same context manager somewhere?

The `chdir` needs to be before `try`.